### ISSUE TITLE
Add a citation link in footer

### ISSUE THREAD

**Issue by keflavich** [camelot-project/frontend#65]
cc @jmdkastro - maybe fill in something temporary for now?  We'll use zenodo to get a DOI.

**Comment by jmdkastro** [comment]
Yes, I'll work on this after 6 June though, as I still have not gotten the actual footer functionality to work. The reason you now see a white bar is that I've included it in the CSS. It's also not "sticky" yet in that its minimum height above the bottom edge of the screen can be negative. I'll fix it, but need to look into it in detail. Given that it's not urgent, I think early June is OK.

Regarding the DOI -- does ADS track such cites? If not, we may not want to do that, as it'll cause confusion as to what to cite. But of course I may be missing something obvious.

**Comment by keflavich** [comment]
@jmdkastro Good question.  I don't think ADS will track DOIs.   Perhaps it is best to leave that empty for now, or put in some sort of placeholder.

**Comment by jmdkastro** [comment]
Good, let's first go with a placeholder then.

**Comment by keflavich** [comment]
Partly addressed in #141
